docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -223,6 +236,11 @@ data
 └── current -> {Release}
 ```
 
+!!! tip "The `data/current` directory"
+    The `data/current` directory is a [symlink](https://en.wikipedia.org/wiki/Symbolic_link) to the most recently-downloaded release directory.
```

Comment:
Trying to make this more accurate. We could add some weasel words here, like "generally" or something about how it might point to test data.
```suggestion
    The `data/current` directory is a [symlink](https://en.wikipedia.org/wiki/Symbolic_link) to the most recent release directory available locally.
```